source/docs/zero-to-robot/step-4/creating-benchtop-test-program-cpp-java.rst

```diff
@@ -84,27 +115,73 @@ Defining the variables for our sample robot
 
 .. tabs::
 
-   .. group-tab:: Java
+   .. group-tab:: PWM
 
-      .. remoteliteralinclude:: https://raw.githubusercontent.com/wpilibsuite/allwpilib/v2022.4.1/wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/gettingstarted/Robot.java
-         :language: java
-         :lines: 19-24
-         :linenos:
-         :lineno-start: 19
+      .. tabs::
 
-   .. group-tab:: C++
+         .. group-tab:: Java
 
-      .. remoteliteralinclude:: https://raw.githubusercontent.com/wpilibsuite/allwpilib/v2022.4.1/wpilibcExamples/src/main/cpp/examples/GettingStarted/cpp/Robot.cpp
-         :language: c++
-         :lines: 12-20
-         :linenos:
-         :lineno-start: 12
+            .. remoteliteralinclude:: https://raw.githubusercontent.com/wpilibsuite/allwpilib/v2022.4.1/wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/gettingstarted/Robot.java
+               :language: java
+               :lines: 19-36
+               :linenos:
+               :lineno-start: 19
 
-      .. remoteliteralinclude:: https://raw.githubusercontent.com/wpilibsuite/allwpilib/v2022.4.1/wpilibcExamples/src/main/cpp/examples/GettingStarted/cpp/Robot.cpp
-         :language: c++
-         :lines: 49-57
-         :linenos:
-         :lineno-start: 49
+         .. group-tab:: C++
+
+            .. remoteliteralinclude:: https://raw.githubusercontent.com/wpilibsuite/allwpilib/v2022.4.1/wpilibcExamples/src/main/cpp/examples/GettingStarted/cpp/Robot.cpp
+               :language: c++
+               :lines: 12-20
+               :linenos:
+               :lineno-start: 12
+
+            .. remoteliteralinclude:: https://raw.githubusercontent.com/wpilibsuite/allwpilib/v2022.4.1/wpilibcExamples/src/main/cpp/examples/GettingStarted/cpp/Robot.cpp
+               :language: c++
+               :lines: 49-57
+               :linenos:
+               :lineno-start: 49
+
+   .. group-tab:: CTRE
+
+      .. tabs::
+
+         .. group-tab:: Java
+
+            .. code-block:: java
+
+               public class Robot extends TimedRobot {
+                  private final WPI_TalonFX m_leftDrive = new WPI_TalonFX(0);
```

Comment:
I don't think it's best practice to have a motor controller on CAN Address 0

Reply:
Why? This has been extremely common AFAIK.

Reply:
ID 0 for motor controllers is not a good idea, and is definitely not the common case -- iirc one of the vendors even doesn't allow setting the ID to 0 and motor controllers are disabled until the ID is changed from the factory default of 0.
So let's not use 0 in the example code.